vmdb/lib/workers/worker_base.rb

```diff
@@ -421,7 +421,8 @@ def parent_is_alive?(parent)
   end
 
   def check_parent_process
-    server = MiqServer.my_server(true)
+    # ensure attributes like "last_heartbeat" are up to date
+    server.reload if server
```

Comment:
So, uh ... @jrafanie found that `check_parent_process` might not be called other than by the specs.  I'll try deleting this in another PR.